ev3dev2/sound.py

```diff
@@ -247,16 +247,27 @@ def play_note(self, note, duration, volume=100, play_type=PLAY_WAIT_FOR_COMPLETE
 
         return self.play_tone(freq, duration=duration, volume=volume, play_type=play_type)
 
-    def play(self, wav_file, play_type=PLAY_WAIT_FOR_COMPLETE):
-        """ Play a sound file (wav format).
+    def play_file(self, wav_file, volume=100, play_type=PLAY_WAIT_FOR_COMPLETE):
+        """ Play a sound file (wav format) at a given volume.
 
         :param string wav_file: The sound file path
+        :param int volume: The play volume, in percent of maximum volume
 
-        :param play_type: The behavior of ``play`` once playback has been initiated
+        :param play_type: The behavior of ``play_file`` once playback has been initiated
         :type play_type: ``Sound.PLAY_WAIT_FOR_COMPLETE``, ``Sound.PLAY_NO_WAIT_FOR_COMPLETE`` or ``Sound.PLAY_LOOP``
 
         :returns: When ``Sound.PLAY_NO_WAIT_FOR_COMPLETE`` is specified, returns the spawn subprocess from ``subprocess.Popen``; ``None`` otherwise
         """
+        if not 0 < volume <= 100:
+            raise ValueError('invalid volume (%s)' % volume)
+
+        if wav_file.endswith(".mp3"):
```

Comment:
Why not require that it _does_ end in `.wav` rather than _not ending_ in `.mp3`?

Reply:
I was worried about breaking things if someone had a wav file that did not end in .wav. That would be pretty silly though...will change it to enforce .wav.

Reply:
I come from the Windows world where file extensions are used for a lot of logic, so maybe my POV is different. wav files without extensions might be common, I'm not sure.